corehq/apps/users/audit/change_messages.py

```diff
@@ -54,33 +54,21 @@ def domain_removal(domain):
             }
         }
 
-    # ToDo: combine this method with the 2 following for two_factor reset
     @staticmethod
-    def registered_devices_reset():
-        return {
-            "devices": {
-                "reset_devices": {}
-            }
-        }
-
-    @staticmethod
-    def two_factor_disabled_for_days(days):
-        return {
-            "two_factor": {
-                "disable_for_days": {"days": days}
-            }
-        }
-
-    @staticmethod
-    def two_factor_disabled_with_verification(verified_by, verification_mode):
-        return {
+    def two_factor_disabled_with_verification(verified_by, verification_mode, devices_reset, disable_for_days):
+        change_message = {
             "two_factor": {
                 "disable_with_verification": {
                     "verified_by": verified_by,
-                    "verification_mode": verification_mode
+                    "verification_mode": verification_mode,
                 }
             }
         }
+        if devices_reset:
+            change_message["two_factor"].update({"reset_devices": {}})
```

Comment:
as mentioned above I don't think there's any value in recording this. Resetting the devices is the mechanism by which we reset 2fa.

Reply:
https://github.com/dimagi/commcare-hq/pull/30253/commits/48cb8fad73224f5f978040fb32736e5a3d99301d